AutomatedTesting/Gem/PythonTests/largeworlds/dyn_veg/EditorScripts/DynVegUtils_TempPrefabCreationWorks.py

```diff
@@ -0,0 +1,91 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+
+def DynVegUtils_TempPrefabCreationWorks():
+    """
+    Summary:
+    An existing level is opened. Each Prefab setup to be spawned by Dynamic Vegetation tests is created in memory and
+    validated against existing test slice components/mesh assignments.
+
+    Expected Behavior:
+    Temporary prefabs contain the expected components/assets.
+
+    Test Steps:
+     1) Open an existing level
+     2) Create each of the necessary temporary Mesh prefabs, and validate the component/mesh setups
+     3) Create the necessary temporary PhysX Collider, and validate the component setup
+     4) Report errors/asserts
+
+    Note:
+    - This test file must be called from the Open 3D Engine Editor command terminal
+    - Any passed and failed tests are written to the Editor.log file.
+            Parsing the file or running a log_monitor are required to observe the test results.
```

Comment:
Just curious, why do we need indention here?

Reply:
Removed this section. Was copy-pasted from another test, but removed to bring in line with other tests.